### PR TITLE
Add `gotty-client` in README.md

### PR DESCRIPTION
**Pull request by moul**
I created [gotty-client](https://github.com/moul/gotty-client) to access my **GoTTY** servers from browser or **command line**

I hope it may be useful for other **GoTTY** users

### CODE REVIEW

**Comment by yudai** [comment]
Hi! Thank you for the PR. Your client is really interesting. Unfortunately, I cannot confirm that it works with the current gotty implementation. I think it's because the latest gotty needs authentication to establish a websocket connection. I'll merge this PR once I confirmed your client works fine with the latest gotty.

**Comment by moul** [comment]
Hi @yudai,

[gotty-client](https://github.com/moul/gotty-client) is now working with the latest version of **GoTTY** (see https://github.com/moul/gotty-client/commit/72597de5c3442f132794cb8e0ff486ad17a44bbb)

---

I didn't find out why, but sometimes I'm getting this error when trying to Read from the websocket:
`WARN[0000] c.Conn.ReadMessage: websocket: unexpected reserved bits 5`
Source: https://github.com/gorilla/websocket/blob/39cd638460acf1c01aaac38effc6e54ee7e04bca/conn.go#L581
Does it mean anything to you?

I should be able to hide this message by adding the reconnect feature, but I would prefer to fix the issue at source if possible

**Comment by yudai** [comment]
@moul  Thanks! Your client works like a charm.

As for the error you are getting, unfortunately, I have no idea about the cause of it.
Gotty have nothing tricky for sending messages with WebSocket, so I didn't come up with any possible causes.
Let me know if you found something bad in gotty.

File: README.md [hunk 1]
```diff
@@ -163,6 +163,10 @@ GoTTY uses [hterm](https://groups.google.com/a/chromium.org/forum/#!forum/chromi
 
 ## Alternatives
 
+### Command line client
+
+* [gotty-client](https://github.com/moul/gotty-client): If you want to connect to an official GoTTY server from your terminal
```
Comment (yudai) [inline README.md]: Could you remove `an official`? Because this repository is official and this word might make readers confused.

Comment (moul) [inline README.md]: Fixed and squashed

**Comment by yudai** [comment]
Thanks a lot! Merged.

**Pull request merged**